go.mod

```diff
@@ -229,3 +229,5 @@ replace (
 )
 
 replace github.com/vishvananda/netlink => github.com/DataDog/netlink v1.0.1-0.20210312173533-6d628a7fc6f3
+
+replace github.com/DataDog/agent-payload => github.com/DataDog/agent-payload v4.65.1-0.20210415170107-6d2b6dc55680+incompatible
```

Comment:
Note: dropping this and pinning to the right version when https://github.com/DataDog/agent-payload/pull/95 gets merged.